Yosemite/Yosemite/Actions/RefundAction.swift

```diff
@@ -7,7 +7,7 @@ import Networking
 public enum RefundAction: Action {
     case createRefund(siteID: Int64, orderID: Int64, refund: Refund, onCompletion: (Refund?, Error?) -> Void)
     case retrieveRefund(siteID: Int64, orderID: Int64, refundID: Int64, onCompletion: (Refund?, Error?) -> Void)
-    case retrieveRefunds(siteID: Int64, orderID: Int64, refundIDs: [Int64], onCompletion: (Error?) -> Void)
+    case retrieveRefunds(siteID: Int64, orderID: Int64, refundIDs: [Int64], deleteStaleRefunds: Bool, onCompletion: (Error?) -> Void)
```

Comment:
I added the flag because in the action `retrieveRefunds` you can specify which refunds to retrieve. So If a consumer chooses just to refresh a few they may not want to delete the ones they didn't fetch.